common/src/main/java/net/opentsdb/data/TimeSeriesStringId.java

```diff
@@ -65,6 +65,8 @@
    */
   public Map<String, String> tags();
 
+  String getTagValue(String key);
```

Comment:
tag public just to be similar and mark final and add java doc about it helping when the underlying structure is not a map.